events/tests/test_event_get.py

```diff
@@ -979,3 +983,36 @@ def test_keyword_and_text(api_client, event, event2, keyword):
     event.save()
     response = get_list(api_client, query_string='combined_text=lapset,aikuiset')
     assert_events_in_response([event], response)
+
```

Comment:
Some tests are still missing and I feel it's quite important to have. For example: `local_ongoing_text`, `local_ongoing_OR`, `local_onging_AND` ... Would be great if the all available filter params are tested properly